source/common/ssl/certificate_validation_context_config_impl.cc

```diff
@@ -56,14 +56,14 @@ CertificateValidationContextConfigImpl::CertificateValidationContextConfigImpl(
 std::vector<envoy::extensions::transport_sockets::tls::v3::SubjectAltNameMatcher>
 CertificateValidationContextConfigImpl::getSubjectAltNameMatchers(
     const envoy::extensions::transport_sockets::tls::v3::CertificateValidationContext& config) {
-  if (!config.match_typed_subject_alt_names().empty() &&
-      !config.match_subject_alt_names().empty()) {
-    throw EnvoyException("SAN-based verification using both match_typed_subject_alt_names and "
```

Comment:
should we keep a warning message for the user? since we ignore this quietly, the user won't know the `match_subject_alt_names` was ignored.

Reply:
Done.